### PR TITLE
Add display configuration options to info addon

### PR DESCRIPTION
**Pull request by z4o4z**
Issue: #1152 

## What I did
Add new options:
- maxPropsIntoLine (default is 3)
- maxPropObjectKeys (default is 3)
- maxPropArrayLength (default is 3)
- maxPropStringLength (default is 50)

## How to test

Run `npm run storybook` to see new stories and test it!

### CODE REVIEW

File: addons/info/src/components/PropVal.js [hunk 1]
```diff
@@ -112,8 +113,14 @@ function previewProp(val) {
 
 export default class PropVal extends React.Component {
   render() {
-    return previewProp(this.props.val);
+    return previewProp(this.props.val, this.context);
```
Comment (ndelangen) [inline addons/info/src/components/PropVal.js]: Can we do it without `context` ?

**Review by ndelangen — CHANGES_REQUESTED** [review]
I don't think we should use context, What problem did you have that you needed context to solve?

**Comment by z4o4z** [comment]
@ndelangen I'm trying to avoid pass identical and never changed props to a lot of places. Why context is  not good idea for it?

File: addons/info/src/components/PropVal.js (hunk 1 above)
Comment (z4o4z) [inline addons/info/src/components/PropVal.js]: @ndelangen fixed

**Comment by z4o4z** [comment]
@ndelangen Hi, of course)

**Review by ndelangen — APPROVED** [review]

**Pull request merged**